src/infohudmodes.asm

```diff
@@ -735,8 +736,16 @@ status_vspeed:
   .newjump
     LDA #$0001 : STA !ram_roomstrat_counter : STA !ram_walljump_counter
 
+    ; Print initial jump speed over item%
+    LDA $0B38 : BNE +
+    LDA $7EC612 : STA $14
+    LDA $0B2D : AND #$0FFF
+    LDX #$0012 : JSR Draw4Hex
+    INC $0B38
```

Comment:
Not a fan of incrementing RAM that the game might use.  What if incrementing the value from 0 to 1 (before the game itself does it) messes with the physics in some way?

I guess I'm asking if we're sure this value is only written to by the game and not read from.

Reply:
I was testing both values, but meant to use $0B1A in the end. $0B2D does get read in a few unknown routines, but I'm confident that $0B1A is only zeroed in vanilla code. I poked through all of the banks in the disassembly with Ctrl+F to confirm, plus breakpoints on memory reads.